flecsi/topo/narray/test/narray.hh

```diff
@@ -18,6 +18,9 @@
 #include "flecsi/topo/narray/coloring_utils.hh"
 #include "flecsi/topo/narray/interface.hh"
 
+// 'interface' is defined as a macro on some platforms
+#undef interface
```

Comment:
Do we have to do this in so many files?

Reply:
I added the `#undef` to all files that actually refer to `interface`, everywhere right after the last #include. That seemed to be the safest way to avoid for it to be redefined. Would you have any suggestions where to place it otherwise?

Reply:
The most we can do for our _clients_ is to put it in the two FleCSI headers that use the identifier directly (`topology_accessor.hh` and `core.hh`).  I'm happy with the various unit tests just `#includ`ing our headers last, since doing so artificially is no uglier than adding the `#undef`.